docs/API.md

```diff
@@ -83,6 +86,9 @@ With the tools array you can ask CE to execute certain tools available for
  For example: ```"tools": [{"id":"clangtidytrunk","args":"-checks=*"}]```
  to execute clang-tidy with all checks enabled.
 
+Libraries can be included by supplying the library id and versions in an array, for example:
+ ```[{"id": "range-v3", "version": "trunk"}, {"id": "fmt", "version": "400"}]```
+
```

Comment:
I wonder if there's a better verb to use than "included". That is, if the user passes `range-v3` here, that doesn't actually `#include` anything into the compiled source file — it just adds some directories to the _include path_.

I wonder if this example should be moved in-line in the sample JSON blob above. Rather than giving the example of `"libraries": []` and then later retrofitting the idea that some array could be for example `[{"id": "range-v3", "version": "trunk"}, {"id": "fmt", "version": "400"}]`, just give the example initially as `"libraries": [{"id": "range-v3", "version": "trunk"}]`!

Pre-existing doc issue: This whole section confusingly mixes the JSON interface with the text (curl-friendly) interface. It would be better if the text interface were explained in a dedicated section. That section could then explain the equivalent of `libraries` for the text interface (which is currently still undocumented, AFAICT).

Reply:
I made an attempt to do these, thanks for the suggestions